Gemfile

```diff
@@ -1,18 +1,32 @@
 source 'https://rubygems.org'
 
-if puppetversion == ENV['PUPPET_GEM_VERSION']
+if puppetversion = ENV['PUPPET_GEM_VERSION']
```

Comment:
This is not a conditional operator.
A single = means it will assign a value to a variable.
I know this code is found else where in other modules, but I'm quite sure it does not do what think it does.
http://www.tutorialspoint.com/ruby/ruby_operators.htm